recipes/libde265/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class Libde265Conan(ConanFile):
+    name = "libde265"
+    description = "Open h.265 video codec implementation."
+    license = "LGPL-3.0-or-later"
+    topics = ("conan", "libde265", "codec", "video", "h.265")
+    homepage = "https://github.com/strukturag/libde265"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "sse": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "sse": True
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
+
+    def _patch_sources(self):
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                              "set(CMAKE_POSITION_INDEPENDENT_CODE ON)", "")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.get_safe("fPIC", True)
+        self._cmake.definitions["DISABLE_SSE"] = not self.options.sse
```

Comment:
I have added `ENABLE_SDL` option for 1.0.9 which will be released soon, please consider adding it for the next version. For now, SDL fails to link to SDL framework on macOS. CI build is ok.

Reply:
Let's add `ENABLE_SDL` in conanfile, but also in upstream CMakeLists for 1.0.8, to avoid finding system installed SDL (which could be linked to `dec265` executable)

Reply:
If we are going to package 1.0.8, yes, disabling SDL build is needed (via patch). In 1.0.9 it will be handled by the option `ENABLE_SDL`.

Reply:
Maybe it's cleaner and easier just to wait for 1.0.9

Reply:
1.0.8 was released yesterday, are you sure 1.0.9 is coming soon?

Reply:
Don't really know. Currently, 1.0.9 is going to include only my two PRs with build fixes, not the functionality or bug fixes. Anyway, we can ask politely.

Reply:
patch added, let's go with 1.0.8